cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/PostgresSinkConsumerIT.java

```diff
@@ -0,0 +1,277 @@
+            TestHelper.execute(insertSql);
+        }
+        Thread.sleep(60000);
+
+    }
+
+    @AfterAll
+    public static void afterClass() throws Exception {
+        Path path_cdcserver = Paths.get("/home/ishaamoncar/test-log.txt");
+        Files.writeString(path_cdcserver, cdcContainer.getLogs(), StandardCharsets.UTF_8);
+
+        Path path_kafka = Paths.get("/home/ishaamoncar/test-log-kafka.txt");
+        Files.writeString(path_kafka, kafka.getLogs(), StandardCharsets.UTF_8);
+
+        Path path_connect = Paths.get("/home/ishaamoncar/test-log-connect.txt");
+        Files.writeString(path_connect, debeziumContainer.getLogs(), StandardCharsets.UTF_8);
+
+        cdcContainer.stop();
+        debeziumContainer.stop();
+        postgreSQLContainer.stop();
+        kafka.stop();
+
+        String dropTableSql = "DROP TABLE test_table";
+        TestHelper.execute(dropTableSql);
+    }
+
+    @Test
+    @Order(1)
+    public void testAutomationOfKafkaAssertions() throws Exception {
+
+        // Verify data on Kafka.
+        try {
+            setKafkaConsumerProperties();
+            consumer.subscribe(Arrays.asList("dbserver1.public.test_table"));
+            long expectedtime = System.currentTimeMillis() + 10000;
+            int flag = 0;
+            Thread.sleep(5000);
+            while (System.currentTimeMillis() < expectedtime) {
+                consumer.seekToBeginning(consumer.assignment());
+                ConsumerRecords<String, JsonNode> records = consumer.poll(15);
+                // assertEquals(records.count(), recordsInserted);
+                System.out.println("Record count: " + records.count());
+                List<Map<String, Object>> kafkaRecords = new ArrayList<>();
+                for (ConsumerRecord<String, JsonNode> record : records) {
+                    ObjectMapper mapper = new ObjectMapper();
+                    if (record.value() != null) {
+                        JsonNode jsonNode = record.value().get("payload");
+                        Map<String, Object> result = mapper.convertValue(jsonNode, new TypeReference<Map<String, Object>>() {
+                        });
+                        kafkaRecords.add(result);
+                    }
+                }
+                Iterator<Map<String, Object>> it = expected_data.iterator();
+                int recordsAsserted = 0;
+
+                for (Map<String, Object> kafkaRecord : kafkaRecords) {
+                    System.out.println("Kafka record " + kafkaRecord);
+                    assertEquals(it.next(), kafkaRecord);
+                    ++recordsAsserted;
+                    if (recordsAsserted == recordsInserted) {
+                        flag = 1;
+                        break;
+                    }
+                }
+                if (flag == 1) {
+                    break;
+                }
+            }
+        }
+        catch (SQLException e) {
+            e.getNextException().printStackTrace();
```

Comment:
`e.printStackTrace()` would do the same thing if I am not wrong. Also throw exception if there's any:

```java
...
catch (SQLException e) {
  e.printStackTrace();
  throw e;
}
```